Source/SwiftyDropbox/Shared/Generated/Files.swift

```diff
@@ -409,7 +410,7 @@ open class Files {
         /// Whether to force the create to happen asynchronously.
         public let forceAsync: Bool
         public init(paths: Array<String>, autorename: Bool = false, forceAsync: Bool = false) {
-            arrayValidator(itemValidator: stringValidator(pattern: "(/(.|[\\r\\n])*)|(ns:[0-9]+(/.*)?)"))(paths)
+            arrayValidator(maxItems: 10000, itemValidator: stringValidator(pattern: "(/(.|[\\r\\n])*)|(ns:[0-9]+(/.*)?)"))(paths)
```

Comment:
do we go through here at all? Not sure if 10k maxItems is an acceptable precondition for us